src/main/java/Todo.java

```diff
@@ -0,0 +1,14 @@
+public class Todo extends Task{
+    protected static String TYPE = "todo";
+    public Todo(String description) {
+        super(description);
+    }
+
```

Comment:
Consider adding @Override for clarity that this is an overridden function from its superclass.